src/helpers/calculate-uptime.ts

```diff
@@ -57,5 +57,6 @@ export const getUptimePercentForSite = async (slug: string): Promise<string> =>
   const downtimeSeconds = await getDowntimeSecondsForSite(slug);
 
   // Return a percentage string
-  return `${Math.max(0, 100 - (downtimeSeconds / totalSeconds) * 100).toFixed(2)}%`;
+  const percentage = Math.max(0, 100 - (downtimeSeconds / totalSeconds) * 100).toFixed(2);
+  return percentage === "100.00" ? "100%" : `${percentage}%`;
```

Comment:
Another way would be to use `parseFloat` like here: https://github.com/aureooms-contrib/graphs/blob/4c2348f4b9a2e37b849bfedc7e9178c7d5bdd953/src/index.ts#L42